superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx

```diff
@@ -68,6 +75,7 @@ interface ActivityProps {
   activeChild: string;
   setActiveChild: (arg0: string) => void;
   activityData: ActivityData;
+  loadedCount: number;
```

Comment:
This should probably be a boolean `isLoaded` rather than the count, unless this component specifically needs the count for some reason.

Reply:
this is a number that is used to determine if all the api calls have finished.